tsuru/client/services.go

```diff
@@ -711,6 +705,21 @@ func (c *ServiceInstanceRemove) Run(ctx *cmd.Context, client *cmd.Client) error
 	return nil
 }
 
+func getErrorAndMessageInSimpleJSON(jsons []string) (string, error) {
```

Comment:
The problem here is that we also need the message from the response and it's not directly in the same json as the error.